### DISCUSSION TITLE
[Console-FastData] Configuration has not been saved

### DISCUSSION THREAD

**Discussion by benedettoraviotta** [mia-platform/community#98] [Q&A]
Hi guys,

I am encountering a problem when saving the console configuration. 

The error occurs when I delete a file from a config map, in this case from the Kafka adapters config map, and then save the configuration. 

The error and the request made to the console are shown in the screen below:
![image](https://user-images.githubusercontent.com/59974565/201722081-68bde7e8-3d69-441a-85f9-8f74302d1fb3.png)

And the response, to the request effected above, is shown below:
![image](https://user-images.githubusercontent.com/59974565/201722283-22f4bc6c-6854-48fb-9e3c-891ec89a657b.png)

The console version is v9.5.2.
The git provider is Atlassian Bitbucket v7.21.0.

**Comment by davidebianchi** [comment]
Hi @benedettoraviotta! Thank you for the report, we will check the issue. @jacoposte we should check the bug and fix it.

The file in the configmap was already created, so have you saved with the file in the configmap and in a second time you deleted it?
At the moment, to bypass the problem, can you create a the file in the repo? The problem is that the file is not found.

**Reply by benedettoraviotta** [reply]
Thanks Davide!

Yes exactly, the error occurs both when I delete the file without replacing it and when I delete the file and create another one (with a different name).

Yes, I confirm that by creating a file, even an empty one, with the same name, the problem is bypassed.
